### ISSUE TITLE
Migration of launchpad issues to GitHub did not import subscribers

### ISSUE THREAD

**Issue by mYnDstrEAm** [mixxxdj/mixxx#10952]
### Feature Description

If possible, please set the same subscribers of Lauchpad issues in GitHub via the email addresses of prior subscribers to issues. Maybe it would then send you an email where you only need to confirm that you'll be subscribed to all the issues that it lists.

It would only work if you used the same email (at least if you don't allow users to link the email used for Lauchpad and the one used for GitHub manually somehow).

This may be an issue with the tool used for migration to GitHub. In that case, don't close this issue before the new issue is created.

Also the GitHub migration styled some texts with code-formatting that weren't code formatted in the original discussion at Lauchpad, as can be seen in #10675.

**Comment by Holzhaus** [comment]
We postponed the migration to GitHub issues for way too long. Hence, we had to write our own migration tool because existing tools didn't work anymore.
I doubt anyone wants to spend additional time one this to write a new  tool and migrate the subscribers.

Subscribers should have received a notification for every Launchpad issue that linked to the corresponding github issue. Just follow the link and subscribe to the GitHub issue.

**Comment by mYnDstrEAm** [comment]
Agree. Cool. In that case, please release the tool as open source on GitHub too. 

Maybe others find it useful if there really is no better current tool for migrating Launchpad issues and if that is the case they may add this functionality. It would be very useful.

**Comment by Holzhaus** [comment]
See #4877.
